docs/docs/contributor/local-setup/ide-setup.mdx

```diff
@@ -6,12 +6,12 @@ sidebar_custom_props:
   icon: TbBrandVscode
 ---
 
-This section will help you set up your IDE for the project. If you haven't set up your development environment, please refer to our [local setup](/contributor/local-setup) section.
+This section will help you set up your IDE for the project. If you haven't set up your development environment, please refer to the [local setup](/contributor/local-setup) section.
 
 
 ## Visual Studio Code
 
-You can use any IDE you want but we recommend using Visual Studio Code as our internal team uses it and we have a lot of extensions and settings that we can share with you.
+You can use any IDE you prefer, but Visual Studio Code is the choice of Twenty's internal team, and has lots of extensions and settings to share with you.
```

Comment:
Remove Twenty. the core team (or internal team good also, let's just remain consistent)